core/vm/operations_acl.go

```diff
@@ -0,0 +1,200 @@
+package vm
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+const (
+	ColdAccountAccessCostEIP2929 = uint64(2600) // COLD_ACCOUNT_ACCESS_COST
+	ColdSloadCostEIP2929         = uint64(2100) // COLD_SLOAD_COST
+	WarmStorageReadCostEIP2929   = uint64(100)  // WARM_STORAGE_READ_COST
```

Comment:
These seem out of place. All our parameters are in `params`. I do get that these belong into a specific EIP, but I'd much rather keep all the number together than scattered all over, lest we run the risk of duplicating some constants and ending up with consensus issues.

Reply:
The naming seems strange? Why is the cold SLOAD but the warm StorageRead? Aren't the two things the same?

Reply:
Yeah, naming is odd. It's in line with what's used in the eip specification though.. 
Re why it's out of place -- I wanted to keep it in one place until we're more certain that it's going in "for real". I'd prefer to scatter it to other places in a follow-up PR once it's decided to activate it.